modules/components/core/org.wso2.carbon.pc.core/src/main/java/org/wso2/carbon/pc/core/internal/ProcessCenterServerHolder.java

```diff
@@ -154,7 +155,7 @@ private void deployAnalyticsDashboard() {
                 String jsonText = new String(content);
                 dashboardResource.setContent(jsonText);
                 dashboardResource.setMediaType("application/json");
```

Comment:
It is better to use the constant MediaType.APPLICATION_JSON  instead of the String literal "application/json"